tachyon/zk/plonk/halo2/snark_verifier_poseidon_transcript.h

```diff
@@ -23,13 +24,14 @@ class SnarkVerifierPoseidonBase {
   using ScalarField = typename AffinePoint::ScalarField;
   using Curve = typename AffinePoint::Curve;
   using CurveConfig = typename Curve::Config;
+  using Params = crypto::BN254PoseidonParams5;
 
   SnarkVerifierPoseidonBase()
       : poseidon_(
             // See
             // https://github.com/scroll-tech/snark-verifier/blob/58c46b7/snark-verifier-sdk/src/param.rs#L7-L10.
-            crypto::PoseidonConfig<ScalarField>::CreateCustom(4, 5, 8, 60, 0)),
-        state_(poseidon_.config) {
+            crypto::PoseidonConfig<Params>::CreateCustom(0)),
+        state_(crypto::SpongeState<Params>()) {
```

Comment:
This lines seems redundant. Please remove it!

Reply:
I believe  `crypto::PoseidonConfig<Params>::CreateCustom(0)),` would still be needed, but I removed ` crypto::PoseidonConfig<Params>::CreateCustom(0)),`.